### PR TITLE
Testing/0.8.4

### PR DESCRIPTION
**Pull request by ericLemanissier**

### CODE REVIEW

**Comment by uilianries** [comment]
@ericLemanissier this build is broken, could you please take a look?

https://travis-ci.com/bincrafters/conan-xkbcommon/jobs/258981515#L1051

**Comment by ericLemanissier** [comment]
the error is a direct consequence of https://github.com/conan-io/conan-center-index/issues/213 https://travis-ci.com/bincrafters/conan-xkbcommon/jobs/258981515#L1062
Do you want me to check meson's deprecation warning anyway ?

**Comment by ericLemanissier** [comment]
@uilianries the error is a direct consequence of conan-io/conan-center-index#213 https://travis-ci.com/bincrafters/conan-xkbcommon/jobs/258981515#L1062
Do you want me to check meson's deprecation warning anyway ?

File: conanfile.py [hunk 1]
```diff
@@ -40,55 +38,16 @@ def configure(self):
...
     def build_requirements(self):
         if not tools.which("meson"):
-            self.build_requires("meson_installer/0.49.0@bincrafters/stable")
+            self.build_requires("meson/0.52.0")
         if not tools.which("bison"):
             self.build_requires("bison_installer/3.2.4@bincrafters/stable")
 
+    def requirements(self):
+        if self.options.with_x11:
+            self.requires("libxcb/1.13.1@bincrafters/stable")
```
Comment (Croydon) [inline conanfile.py:50]: I'm confused by the list of system_requirements vs this

xkb-data/xkeyboard-config IS included in libxcb as well, but the system packages libxcb-xkb-dev/libxcb1-dev/libxcb-devel to NOT include the system packages xkb-data/xkeyboard-config?

Comment (ericLemanissier) [reply conanfile.py:50]: xkb-data and xkeyboard-config are not included in libxcb, but it's not strictly speaking mandatory to build xkbcommon : https://github.com/xkbcommon/libxkbcommon/blob/master/PACKAGING#L17. This is why I had to do https://github.com/bincrafters/conan-xkbcommon/pull/2/files#diff-2b1d42f71f22b7ea0412d7602dec166fR90.
A better solution would be to create a conan package for xkeyboard-config in the x11 project, and require it.

**Pull request merged**